### PR TITLE
LPS-84822 Wrap XSLURIResolver if the secure processing feature is activated

### PR DESCRIPTION
**Pull request by martamedio**

### CODE REVIEW

**Comment by martamedio** [comment]
ci:test:sf

**Comment by martamedio** [comment]
ci:test:relevant

**Comment by martamedio** [comment]
ci:test:sf

**Comment by martamedio** [comment]
ci:test:relevant

File: modules/apps/portal-template/portal-template-xsl/src/main/java/com/liferay/portal/template/xsl/internal/XSLSecureURIResolver.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+				_log.error(
+					StringBundler.concat(
+						"Denied access to resource: ", href,
+						". Access to local network is disabled by the secure ",
+						"processing feature."));
+
+				return null;
+			}
+
+			return _xsluriResolver.resolve(href, base);
```
Comment (topolik) [inline modules/apps/portal-template/portal-template-xsl/src/main/java/com/liferay/portal/template/xsl/internal/XSLSecureURIResolver.java]: Here I expect NPE.

Comment (martamedio) [reply modules/apps/portal-template/portal-template-xsl/src/main/java/com/liferay/portal/template/xsl/internal/XSLSecureURIResolver.java]: It is being verified that it is not null at XSLTemplate#110

Comment (topolik) [reply modules/apps/portal-template/portal-template-xsl/src/main/java/com/liferay/portal/template/xsl/internal/XSLSecureURIResolver.java]: Ah, so it means when there is no custom URI resolver we allow all connections?

Comment (martamedio) [reply modules/apps/portal-template/portal-template-xsl/src/main/java/com/liferay/portal/template/xsl/internal/XSLSecureURIResolver.java]: But... I think it's more legible to make the logic as you're saying, I change it

**Comment by topolik** [comment]
Pull request submitted at: https://github.com/tinatian/liferay-portal/pull/1180

View just my changes: https://github.com/topolik/liferay-portal/compare/martamedio:a5a68815d9...9c935a8725
View total diff: https://github.com/topolik/liferay-portal/compare/55499ba696...9c935a8725

**Pull request closed without merging**